docs/changelog/1984.misc.rst

```diff
@@ -0,0 +1 @@
+Enable building tox with ``setuptools_scm`` 6+ by :user:hroncok
```

Comment:
@hroncok you forgot the backticks after `:user:` :point_up:

Reply:
Oh, not again. If only a machine could tell me this before this was merged :thinking:

Reply:
https://github.com/tox-dev/tox/pull/1989

Reply:
*/me needs to teach chronographer parse RST maybe*

Reply:
Even simpler, https://github.com/tox-dev/tox/blob/master/.pre-commit-config.yaml#L56-L66 can have a simple regex rule in there 🤷🏻

Reply:
@gaborbernat I also thought so and I'm trying to explore this. But then I started to think about cases when the person doesn't want to mark themselves as an author and it got rather complicated...

Reply:
It doesn't have to be a must-have field, but just check for typos with it 🤔

Reply:
Yeah.. You're right. I overengineered it, I guess: https://github.com/tox-dev/tox/pull/1990.